### PR TITLE
:zap: Add Summary for CICERO

### PR DESCRIPTION
**Pull request by maanavgoyal**

### CODE REVIEW

File: summaries/CICERO.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Meta Fundamental AI Research Diplomacy Team (FAIR), Antin Bakhtun, Noam Brown, Emily Dinan **2022**
+
+## Summary
+
+Unlike competitive games like Chess and Go, which computers have mastered through self-play, the game Diplomacy has been a different challenge because it involves major human elements
+like cooperation and understanding other players' motivations. CICERO is the first AI agent to achieve human-level performance in Diplomacy.<br>
+CICERO uses a language model integrated with a strategy model by understanding conversations happening during the game and generating dialogue to achieve its objectives.
+
+##Diplomacy
```
Comment (ChanBong) [inline summaries/CICERO.md:11]: add space to make this a heading

File: summaries/CICERO.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Human-level play in the game of Diplomacy by combining language models with strategic reasoning
+
+Meta Fundamental AI Research Diplomacy Team (FAIR), Antin Bakhtun, Noam Brown, Emily Dinan **2022**
```
Comment (ChanBong) [inline summaries/CICERO.md:3]: Change order. See [template](https://github.com/vlgiitr/papers_we_read/blob/master/Summary_Template.md)

**Review by ChanBong — CHANGES_REQUESTED** [review]

File: summaries/CICERO.md (hunk 2 above)
Comment (maanavgoyal) [reply summaries/CICERO.md:3]: Do you mean putting Meta team name after the authors' names? That is not the conference name if you misunderstood it for that.
They have stated it in the paper in this manner only, where they have first stated the team name and then the authors.

File: summaries/CICERO.md (hunk 1 above)
Comment (maanavgoyal) [reply summaries/CICERO.md]: done

File: summaries/CICERO.md (hunk 2 above)
Comment (ChanBong) [reply summaries/CICERO.md:3]: Oh my bad ! Misunderstood that 😅

**Review by ChanBong — APPROVED** [review]

**Comment by ChanBong** [comment]
I just noticed that images are not visible in summary. 
I checked and seems like you have made a typo, writing `PNG` instead of correct `jpg`

Kindly fix

**Pull request merged**